### PR TITLE
Split core function from user interface

### PR DESCRIPTION
**Pull request by suzizecat**
Hi,

This merge request aim to split the mergeLef file into a function (that might be imported in other python scripts) and a user interface (parameter parsing) used when the file is used as standalone. 

This could allow other flow tools interacting with OpenROAD to easily use mergeLef.

Best regards,
Julien FAUCHER

### CODE REVIEW

File: flow/util/mergeLef.py [hunk 1]
```diff
@@ -6,81 +6,83 @@
...
-# Parse and validate arguments
-# ==============================================================================
-parser = argparse.ArgumentParser(
-    description='Merges lefs together')
-parser.add_argument('--inputLef', '-i', required=True,
-                    help='Input Lef', nargs='+')
-parser.add_argument('--outputLef', '-o', required=True,
-                    help='Output Lef')
-args = parser.parse_args()
+def mergeLef(inputLef,outputLef) :
```
Comment (maliberty) [inline flow/util/mergeLef.py]: Please put a space after the comma per PEP8

File: flow/util/mergeLef.py [hunk 2]
```diff
@@ -6,81 +6,83 @@
...
+  # Parse and validate arguments
+  # ==============================================================================
+  parser = argparse.ArgumentParser(
+      description='Merges lefs together')
+  parser.add_argument('--inputLef', '-i', required=True,
+                      help='Input Lef', nargs='+')
+  parser.add_argument('--outputLef', '-o', required=True,
+                      help='Output Lef')
+  args = parser.parse_args()
+  mergeLef(args.inputLef,args.outputLef)
```
Comment (maliberty) [inline flow/util/mergeLef.py]: Please put a space after the comma per PEP8

**Review by maliberty — CHANGES_REQUESTED** [review]
I'm curious what other purpose you intend to use this script for.

**Comment by suzizecat** [comment]
The purpose might be easier interfacing when using SiliconCompiler since it looks like OpenROAD needs tlef and lef to be merged. Therefore it would be easier to be able to just import the the mergeLef function rather than spawning a whole new process for this.

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
The mergeLef is a holdover from standalone TritonRoute.  It isn't needed anymore or used afaik.

**Comment by rovinski** [comment]
Correct, MergeLef isn't used anywhere in the flow. I thought it was already removed.

**Comment by tspyrou** [comment]
@suzizecat can you merge in the changes in master to your branch. Hopefully that will fix the CI failure.

**Comment by maliberty** [comment]
@tspyrou I don't see any reason to merge this - we should just remove mergeLef as it is obsolete

**Comment by maliberty** [comment]
Removed mergeLef in https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/484

**Pull request closed without merging**